docs/source/pages/howToWorkWithPlatform/mainApplication/templateList/templateManagement/documentManagementOnTemplate/structuredDocument/fields/table/table.rst

```diff
@@ -4,8 +4,7 @@ Table Field
 
 .. toctree::
 
-Table field allows to customer to add dynamic data (such as text, numeric, currency, date and time, e-ink signature, dictionary and lookup fields.) to the document. 
-Also allows to prepare table exactly of the size which needed for participant of the envelope processing flow (currently restricted by 5 columns and doesn't restricted by rows quantity).
+Table field allows user to add dynamic data to the document. Also it allows to prepare table of exact needed size for participant of the envelope processing flow.
```

Comment:
Please add table restriction information such as max columns quantity, min column width and etc. This info should be added to table description that user be able to read it without any issues